### ISSUE TITLE
Angular/Cordova ou Ionic?

### ISSUE THREAD

**Issue by wgbn** [lfdev-gdg/evento-lfdev-checkin-app#1]
Pessoal,

Estou na dúvida sobre que tecnologia usar aqui.
**Angular 2 + Cordova puro**
ou
**Ionic 2**

E ai?

**Comment by fxcosta** [comment]
Da pra responder issue por email? Rs. Enfim, eu curto bastante o ionic.
Sempre foi minha opção pra mobile. Acho que usar angular 2 + cordova puro
vai fazer você chegar quase num ionic de qualquer jeito, não?

Em 03/03/2017 09:14, "Walter Gandarella" <notifications@github.com>
escreveu:

**Comment by wgbn** [comment]
Sim, mas é que familiarizei tanto com o sistema de rotas do Angular 2 que o Ionic não incluiu que fiquei realmente na dúvida

**Comment by fxcosta** [comment]
Hum, entendi. Uma opção não é usar o angular pra criar um pwa não? Esse
chekin é como?

Em 03/03/2017 10:09, "Walter Gandarella" <notifications@github.com>
escreveu:

**Comment by wgbn** [comment]
A inscrição gera um QR-code único pro usuário.
A ideia é um sisteminha onde na chegada do evento, a gente somente escaneie o QR e automaticamente marque que tal pessoa está presente, isso a gente usa para gerar os certificados pra quem esteve presente e para, se rolar que nem sei ainda, fazer o sorteio de brindes.

**Comment by fxcosta** [comment]
Então ficaria como vc achar melhor, já que você vai dar o start. Eu votaria em pwa ou ionic, até porque se for só um scanner de qrcode, é melhor pra experiência dele e você ainda pode usar como case pra sua palestra!

**Comment by dannyserena** [comment]
Concordo com @fxcosta, como só vi ionic 1 fica a seu critério. Agora em termos de praticidade pra scanear o QR não seria melhor usar ionic?
